Resources/Prototypes/Entities/Objects/Materials/Sheets/glass.yml

```diff
@@ -48,6 +48,12 @@
     solutions:
       glass:
         canReact: false
+  # Estacao Pirata - IPC Healing
+  - type: BlindHealing
+    damageContainers: 
+      - Silicon
+  - type: StackPrice
```

Comment:
remove this here and in the system, no reason to couple economy to healing (and this value probably ignored mining rebalance anyway)